README.md

```diff
@@ -1,8 +1,34 @@
 ember-class-constructor-fix
 ==============================================================================
 
-[Short description of the addon.]
+This addon fixes a common bug that occured in versions of Ember prior to 3.13,
+where if a particular combination of browser targets was used, it was possible
+to have failures when attempting to zebra-stripe native and Classic class
+definitions:
 
+```js
+class Foo extends EmberObject {}
+
+const Bar = Foo.extend();
+
+class Baz extends Bar {}
+
+const Qux = Baz.extend();
+```
+
+Zebra-striping here refers to the fact that we are extending using native
+syntax, then classic syntax, then native syntax again.
+
+The bug came down to Ember using a precompiled build that targeted legacy
+browsers such as IE11. In some cases, it was possible that Ember would use this
+build, even though _classes_ didn't need to be transformed. The user's classes
+would not be transformed, but one particular class in Ember would - the class
+that is created in `EmberObject.extend()`. Native classes _cannot_ interop with
```

Comment:
Can you clarify what this sentence means? I'm not following what "standard functions" refers to here, so it's confusing.